### ISSUE TITLE
Placement of user.el in new Scimax 3 configuration

### ISSUE THREAD

**Issue by Arjay-El** [jkitchin/scimax#476]
I really liked the use of user,el an want to retain its contents. I have tried adding a line to init.el and also packages.el but that has not produced the result I want (file being read). Do you have a suggestion as to where, in your loading sequence, I should call user.el?

My config: I am on Win11. I clone your repo to c:/scimax then create an emacs-win directory and copy Emacs 28.2 into it. I then run scimax.bat a few times until the errors & warnings reduce to a reasonable number.

**Comment by jkitchin** [comment]
I use something like this in ~/.emacs.d/init.el

```
(load  "/Users/jkitchin/Dropbox/emacs/user/preload.el")
(load  "/Users/jkitchin/Dropbox/emacs/scimax/init.el")
(load  "/Users/jkitchin/Dropbox/emacs/user/user.el")
```

Then I make sure the emacs I want is on the executable path. That way I simply run the emacs command in a terminal and it launches scimax.

**Comment by JSpenced** [comment]
I was going to have a similar question, so I will post it here. I noticed that you removed loading the `user.el` in this commit (`ac8d159 * try removing user dir`). My `user.el` loads a couple other files as well in the same folder and older packages from Drew Adams that aren't in Melpa. It seems those all have to reference the full path now.

Is it recommended to have an `init.el` in `emacs.d` like the above and load both `init.el` of your package and `user.el` there? Then, I think I will need to change my `user.el` to have full paths to load `functions.el` and any requires will also need the full path?

Also, if you have a link to the recommended setup, that would be great! I didn't know scimax 3 was released as mentioned above, so I will check it out.
